### ISSUE TITLE
Please create a simple interactive .sh script to automate installtion

### ISSUE THREAD

**Issue by ghost** [sickcodes/droid-native#3]
It would be nice and easier to have a simple .sh script that would install the required kernel with all binders 
Then check if waydroid and its image  is installed , if yes then skip if no then install 
and install all the remaining things .
So as a newbie user just has to run the script and get working waydroid on arch or arch based systems
In worst case , system will not boot which can be restored using timeshift or any other backup

**Comment by sickcodes** [comment]
Yeah we are just waiting on new binder and ashmem modules, and Ill put this in the AUR

**Comment by ghost** [comment]
Due to no development or updates on this issue.
I am closing this issue.
